### PR TITLE
Interactive terminal tutorial

### PR DESCRIPTION
**Pull request by gregersrygg**
Content for the interactive terminal tutorial. Put the part about Arduino as serial forwarder into it's own guide so the interactive terminal tutorial becomes much shorter.

### CODE REVIEW

File: docs/tutorials/interactive-terminal.md [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+  - primary
+tags:
+  - tutorial
+  - terminal
+  - ee-nbiot-01
+---
+
+## TODO
+- [ ] Update link to "See own guide on low power"
+- [ ] Add link to how to set up a device in the NB-IoT Developer Platform
```
Comment (pkkummermo) [inline docs/tutorials/interactive-terminal.md]: Remove these and add as issues

File: docs/tutorials/interactive-terminal.md [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+### Install serial adapter driver
+Most cables require you to install a driver for the USB serial adapter. Check the product description and it should guide you to the correct driver for your operating system.
+
+### Serial terminal application
+You can use any serial terminal application you like. We'll only explain how to use CoolTerm in this guide - since it's free, easy to use and works on Windows, macOS and Linux.
+
+1. [Download and install CoolTerm for your OS](http://freeware.the-meiers.org/)
+1. Open app and click the «Options» button
+    ![CoolTerm Serial Port options screenshot](img/coolterm-serialport.png)
+    ![CoolTerm Terminal options screenshot](img/coolterm-terminal.png)
```
Comment (pkkummermo) [inline docs/tutorials/interactive-terminal.md]: The images seems to misalign a bit

**Comment by pkkummermo** [comment]
Remove the `Arduino tutorials` entry in the menu as the new tutorials effectively replace that :)

File: docs/tutorials/interactive-terminal.md (hunk 2 above)
Comment (gordonklaus) [inline docs/tutorials/interactive-terminal.md]: Move this image under step 5.

File: docs/tutorials/interactive-terminal.md [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+
+### Install serial adapter driver
+Most cables require you to install a driver for the USB serial adapter. Check the product description and it should guide you to the correct driver for your operating system.
+
+### Serial terminal application
+You can use any serial terminal application you like. We'll only explain how to use CoolTerm in this guide - since it's free, easy to use and works on Windows, macOS and Linux.
+
+1. [Download and install CoolTerm for your OS](http://freeware.the-meiers.org/)
+1. Open app and click the «Options» button
+    ![CoolTerm Serial Port options screenshot](img/coolterm-serialport.png)
```
Comment (gordonklaus) [inline docs/tutorials/interactive-terminal.md]: Maybe move this image under step 3.

**Comment by pkkummermo** [comment]
Increase `$docs-width-content--desktop` to 1024 for better table handling. Still kinds sucks on smaller devices, but that's tables for you.

**Review by pkkummermo — APPROVED** [review]

File: docs/tutorials/interactive-terminal.md [hunk 4]
```diff
@@ -0,0 +1,199 @@
...
+--------------:|------------
+VCC +5V/3.3V - red | VCC
+GND - black | GND
+RX - yellow | RXD
+TX (**3.3V**) - orange | TXD
+
+[![FTDI to EE-NBIOT-01](img/ftdi-small.jpg)<br/>
+See image in full resolution](img/ftdi.jpg)
+
+## Terminal commands
```
Comment (gordonklaus) [inline docs/tutorials/interactive-terminal.md:100]: Can you add an anchor here?  The Raspberry Pi serial tutorial wants to link to this.

Comment (gordonklaus) [reply docs/tutorials/interactive-terminal.md:100]: Call it `terminal-commands` if you would be so kind.

Comment (gregersrygg) [reply docs/tutorials/interactive-terminal.md:100]: That's done automagically from the title ;)
[link](http://localhost:8888/tutorials/interactive-terminal.html#terminal-commands)

Comment (gordonklaus) [reply docs/tutorials/interactive-terminal.md:100]: Nice :)  Too bad you can't see the anchor when you hover..

Comment (gregersrygg) [reply docs/tutorials/interactive-terminal.md:100]: True, but viewing the source and checking for an id attribute always works.

Comment (gordonklaus) [reply docs/tutorials/interactive-terminal.md:100]: It's just nice for normal users (peasants) to be able to link to sections.

**Pull request merged**